# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/wgpu-core/src/track/range.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/wgpu-core/src/track/range.rs b/third_party/rust/wgpu-core/src/track/range.rs
rename from gfx/wgpu/wgpu-core/src/track/range.rs
rename to third_party/rust/wgpu-core/src/track/range.rs
--- a/gfx/wgpu/wgpu-core/src/track/range.rs
+++ b/third_party/rust/wgpu-core/src/track/range.rs
@@ -1,12 +1,10 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
+//Note: this could be the only place where we need `SmallVec`.
+//TODO: consider getting rid of it.
 use smallvec::SmallVec;
 
 use std::{cmp::Ordering, fmt::Debug, iter, ops::Range, slice::Iter};
 
 /// Structure that keeps track of a I -> T mapping,
 /// optimized for a case where keys of the same values
 /// are often grouped together linearly.
 #[derive(Clone, Debug, PartialEq)]